traefik_mesh/README.md

```diff
@@ -69,11 +69,10 @@ Need help? Contact [Datadog support][9].
 [2]: https://app.datadoghq.com/account/settings/agent/latest
 [3]: https://docs.datadoghq.com/agent/kubernetes/integrations/
 [4]: https://github.com/DataDog/integrations-core/blob/master/traefik_mesh/datadog_checks/traefik_mesh/data/conf.yaml.example
-[5]: https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent
```

Comment:
Not sure how you found that this lines the problem, but approving to fix the issue.

Reply:
also do the numbers have to updated if your removing `5`, theres a gap now?